app/jobs/report_usage_job.rb

```diff
@@ -0,0 +1,7 @@
+class ReportUsageJob < ApplicationJob
+  queue_as :default
```

Comment:
Should we throw something into the README, to clarify that usage reporting is experimental?